fastlane/Fastfile

```diff
@@ -526,8 +526,8 @@ platform :ios do
 
   desc 'Downloads localized strings and App Store Connect metadata from GlotPress'
   lane :download_localized_strings_and_metadata_from_glotpress do
-    download_localized_strings_and_metadata
-    download_localized_app_store_metadata_from_glotpress
+    #download_localized_strings_and_metadata
```

Comment:
@mokagio to document here on Github: I removed these calls because there are a lot of warnings due to keys that are not present on the base locale:

```
`siri_intent_definition_key_0EmR82` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_0YXvyS` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_1uT6VF` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_3aWmex` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_5yWeuV` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_bVm1vW` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_cfJexL` was unexpected, as it is not present in the base locale.
`siri_intent_definition_key_cV2HLF` was unexpected, as it is not present in the base locale.
```

Reply:
I've been looking at this. 

I think the issue was indeed due to the localization tooling being half backed / half used.

Those keys have been extracted from the `en.lproj/Localizable.strings` but they must still have been in the other languages versions that had previously been downloaded.

I tried `bundle exec fastlane generate_strings_file_for_glotpress && bundle exec fastlane  download_localized_app_store_metadata_from_glotpress && bundle exec fastlane lint_localizations` on `trunk` and they all passed.